packages/react-native-editor/CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ For each user feature we should also add a importance categorization label  to i
 
 ## 1.52.0
 
+## 1.52.1
+
+- [*] Fixes for the generated localized strings files.
+
+## 1.52.0
+
```

Comment:
Looks like we duplicated the entry for `1.52.0`.

Reply:
No worries, let me know if you create the PR. I could also update the file if you’re working on other tasks.

Thank you very much @dcalhoun for wrangling the release ❤️, you did an amazing job!